client/src/components/CalendarHeader/CalendarHeader.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { FiMenu } from "react-icons/fi";
+import { MdClose } from "react-icons/md";
+import styles from "./CalendarHeader.module.css";
+
+const CalendarHeader = () => {
+  const [menuOpened, setMenuOpened] = useState(false);
+
+  return (
+    <div className={styles.calendarHeaderContainer}>
+      <button
+        className={styles.navButton}
+        onClick={() => {
+          setMenuOpened(!menuOpened);
+        }}
+      >
+        {!menuOpened ? (
+          <FiMenu className={styles.hamburgerIcon} />
+        ) : (
+          <MdClose className={styles.hamburgerIcon} />
+        )}
+      </button>
+      <p className={styles.calendarPageHeader}>Daily Tasks</p>
```

Comment:
Minor comment: Perhaps on a desktop view, it might be a bit better to have the "Daily Tasks" title centered
![image](https://user-images.githubusercontent.com/24628243/156702377-b9370732-4ab2-4be3-a7b3-15467e06874a.png)

If you wanted to take this approach: This can be done with media queries, where after a certain screensize, it can be centered

Reply:
To be implemented when together with other desktop layouts.